src/system/unwind.c

```diff
@@ -155,6 +167,21 @@ void report_data_abort(uint32_t _sp) {
 	fprintf(stderr, "\t%p\n", (void*)vrs.core.r[R_PC]);
 	__gnu_Unwind_Backtrace(trace_fn, NULL, &vrs);
 	fputs("END OF TRACE\n", stderr);
+	
+	for(size_t i = 0; i < trace.size / 4; i++) {
+		vexDisplayString(brain_line_no++, "%p %p %p %p", (void*)trace.pcs[4*i], (void*)trace.pcs[4*i+1], (void*)trace.pcs[4*i+2], (void*)trace.pcs[4*i+3]);
```

Comment:
This display string should probably have more formatting than just 4 raw pointers in a line. There's currently not even any indication to the user that it's a stack trace.

Reply:
I wanted to include the word "STACKTRACE: " before the pointers, but just eyeballing it based on pictures of the current data abort display, I wasn't sure if the text would be too wide to fit on the screen.

Reply:
Added "TRACE: " before the pointers to indicate that its a stacktrace.